python-requirements.txt

```diff
@@ -4,11 +4,17 @@
 
 # Keep sorted.
 bokeh
+codetiming
+more-itertools
+networkx
 pycryptodome
 pyyaml
+ray
 scared
 tqdm
 trsfile
+# can be removed after switching to ray
+joblib
```

Comment:
Is this still needed?

Reply:
Yes, this is required by [this line](https://github.com/lowRISC/ot-sca/blob/8e61b89075ae5a886d35090b2675d721705d495c/cw/cw305/correlation-enhanced_collision_attack.py#L18) in `correlation-enhanced_collision_attack.py`.